src/platform/packages/shared/shared-ux/document_data_cascade/impl/src/lib/parse_esql.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { Parser } from '@kbn/esql-ast';
+import {
+  isESQLAstBaseItem,
+  isESQLFunction,
+  type ESQLColumn,
+  type ESQLCommandOption,
+} from '@kbn/esql-ast/src/types';
+
+const isESQLCommandOption = (node: unknown): node is ESQLCommandOption =>
```

Comment:
I think I have done comments about this file in another PR 🤔 Hadn't we agreed that we will change the implementation?